lib/msf/base/serializer/readable_text.rb

```diff
@@ -51,34 +51,64 @@ def self.dump_module(mod, indent = "  ")
   # @return [String] the string form of the table.
   def self.dump_exploit_targets(mod, indent = '', h = nil)
     tbl = Rex::Text::Table.new(
-      'Indent'  => indent.length,
       'Header'  => h,
       'Columns' =>
         [
+          'IsTarget',
           'Id',
           'Name',
-        ])
+        ],
+      'SortIndex' => 1,
+      'ColProps' => {
+        'IsTarget' => {
+          'Stylers' => [Msf::Ui::Console::TablePrint::RowIndicatorStyler.new],
+          'ColumnStylers' => [Msf::Ui::Console::TablePrint::ColumnHeaderAndDecorationStyler.new],
+          'MaxWidth' => 2
+        }
+      }
+    )
 
-    mod.targets.each_with_index { |target, idx|
-      tbl << [ idx.to_s, target.name || 'All' ]
-    }
+    mod.targets.each_with_index do |target, idx|
+      if mod.target == target
+        indicator_row = true
+      else
+        indicator_row = false
+      end
+
+      tbl << [indicator_row, idx.to_s, target.name || 'All' ]
+    end
 
     tbl.to_s + "\n"
   end
 
   def self.dump_evasion_targets(mod, indent = '', h = nil)
     tbl = Rex::Text::Table.new(
-      'Indent'  => indent.length,
       'Header'  => h,
       'Columns' =>
         [
+          'IsTarget',
           'Id',
           'Name',
-        ])
+        ],
+      'SortIndex' => 1,
+      'ColProps' => {
+        'IsTarget' => {
+          'Stylers' => [Msf::Ui::Console::TablePrint::RowIndicatorStyler.new],
+          'ColumnStylers' => [Msf::Ui::Console::TablePrint::ColumnHeaderAndDecorationStyler.new],
```

Comment:
We could probably just make this a generic reusable component that doesn't output the column header by default:

```suggestion
          'ColumnStylers' => [Msf::Ui::Console::TablePrint::OmitColumnHeader.new],
```

The implementation would be updated to no longer be hard coded to `IsTarget` as well

Reply:
This seems reasonable, another path might be to be more specific. Consider `IndicatorColumnStyler` that instead of a single `IsTarget` set can evaluate some block to determine `true` or `false` for each row so you could mark multiple rows as `selected` in other use cases.